c10/core/MemoryFormat.h

```diff
@@ -126,20 +127,20 @@ inline std::vector<int64_t> get_channels_last_strides_3d(IntArrayRef sizes) {
 // 3. All helper functions have similar comments, only 1st helper function is
 // commented here.
 template <typename T>
-inline bool is_channels_last_strides_2d_s4(
+inline bool is_channels_last_strides_2d_s4_or_false(
```

Comment:
how hard would it be to do a 
sym_is_channels_last_strides_2d_s4
sym_is_channels_last_strides_2d

mm actually I am I am ok with this for now because
```
SymBool SymbolicShapeMeta::compute_channels_last_2d_dim5() const {
  init_is_channels_last_3d_contiguous();
  if (guard_or_false(is_channels_last_3d_contiguous(), __FILE__, __LINE__)) {
    return false;
  }
  return ~is_channels_last_3d_contiguous() &
      compute_strides_like_channels_last_2d();
}
```

already have guard_or_false here so it wont make an effect ,

but ideally this should be cleaned up to follow contiguous model, were only users can guard_or_false we do not implicitly  do it in APIs that do not have that naming.

can you add a comment and create gh issue to track this.